<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>iconchat.ico</normaloff>iconchat.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:24px;
}
QListWidget
{
background:#333;
color:#717072;
}
QFrame
{
background:#333		;
border-radius:15px;
}
QPushButton
{
background:#F08080	;;
border-radius:15px;
}
QToolButton
{
background:#F08080	;;
border-radius:60px;
}
QLabel
{
color:white;
font-size:18px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#ffffff	;
}
QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}
QPushButton#back
{
border-radius:20px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget
{
background:white;
}</string>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="users_online">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget
{
background-color:white;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
